generators/statistics.js

```diff
@@ -0,0 +1,121 @@
+const uuid = require('uuid/v1');
+const Config = require('conf');
+const osLocale = require('os-locale');
+const axios = require('axios');
+const os = require('os');
+
+const DO_NOT_ASK_LIMIT = 100;
+
+const API_PATH = 'https://start.jhipster.tech/api';
+
+class Statistics {
+    constructor() {
+        this.config = new Config({
+            configName: 'jhipster-insight',
+            defaults: {
+                clientId: uuid(),
+                doNotAskCounter: 0
+            }
+        });
+        this.statisticsAPI = API_PATH;
+        this.clientId = this.config.get('clientId');
+        this.doNotAskCounter = this.config.get('doNotAskCounter');
```

Comment:
This shouldnt be per project, this should be saved in the a global config store. Is `new Config` creating a global store in users home or something? dow it work with winodws/linux/mac?

Reply:
https://github.com/sindresorhus/conf

Reply:
It's a global config that is used by yeoman. It's supposed to work pretty much on every common OS.

https://github.com/sindresorhus/conf

Reply:
ok